tests/unit/shared/test_get_dist.py

```diff
@@ -0,0 +1,18 @@
+from vic import lib as vic_lib
+
+
+def test_get_dist():
```

Comment:
I'm a little concerned with this test mostly using 0 as the base latitude and longitude. Try some larger positive and negative numbers.

What is supposed to happen for lat > 90 or lat < -90 and lon < -180 and lon > 360 etc.